t/local/get.t

```diff
@@ -1,87 +1,75 @@
-#
-# Test retrieving a file with a 'file://' URL,
-#
-
-if ($^O eq "MacOS") {
-    print "1..0\n";
-    exit;
-}
-
+use strict;
+use warnings;
+use Test::More;
 
 use File::Temp 'tempdir';
+use LWP::Simple;
+require LWP::Protocol::file;
 
-# First locate some suitable tmp-dir.  We need an absolute path.
-$TMPDIR = undef;
-for (tempdir()) {
-    if (open(TEST, ">$_/test-$$")) {
-        close(TEST);
-	unlink("$_/test-$$");
-	$TMPDIR = $_;
-	last;
-    }
+my $TMPDIR = undef;
+
+if ( $^O eq 'MacOS' ) {
+    plan skip_all => 'Cannot test on this platform';
```

Comment:
MacOS at this point is identified as 'darwin'.  MacOS is for Mac OS 7, 8, and 9.  I don't know the reasoning behind not allowing the tests on older macs, but maybe this is no longer necessary.